en/08_Changelogs/6.0.0.md

```diff
@@ -716,11 +716,32 @@ If you reference any of these classes in your project or module, most likely in
 
 MySQL 5.6 and 5.7 are no longer supported. The minimum supported version is MySQL 8.0. We support and test against the latest LTS releases of MySQL and MariaDB.
 
+### MySQL now defaults to `utf8mb4` {#mysql-utf8mb4}
```

Comment:
Please add an entry to the TOC at the top

Reply:
Done.